test/unit/media.html5.js

```diff
@@ -71,6 +71,19 @@ test('test playbackRate', function() {
   strictEqual(tech.playbackRate(), 0.75);
 });
 
+test('should remove the controls attribute when recreating the element', function() {
```

Comment:
I am not sure why the test is failing on iPhones. Are you testing with 8.1? I feel like this passed on earlier versions but I'll have to go dig up a device to check.